python_modules/dagster/dagster/_core/launcher/base.py

```diff
@@ -69,7 +69,7 @@ def launch_run(self, context: LaunchRunContext) -> None:
         """
 
     @abstractmethod
-    def terminate(self, run_id: str) -> bool:
+    def terminate(self, run_id: str, message=None) -> bool:
```

Comment:
do we write this to the event log somewhere?

Reply:
artifact from a bygone implementation